### ISSUE TITLE
[a11y] 11.2.5.1 Pointer gestures

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1339]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

### Image Viewer

- [x] The carousel elements (images) can only be moved using swipe gestures. There is
There is no alternative input option using simple pointer gestures.

**Fixed in #1453 by adding previous/next buttons that become visible when hovered with a trackpad or mouse pointer.**
